config/config.ini

```diff
@@ -2043,6 +2043,9 @@ name = Shannon -jj Behrens
 [http://shiningpanda.com/feeds/all.atom.xml]
 name = ShiningPanda
 
+[https://tech.shopkick.com/tagged/python/atom.xml]
+name = Shopkick Tech Blog
```

Comment:
Hi Tres! I actually did run sort-ini.py and that's where it ended up, funny enough. The rest of the file does not seem sorted by URL, either, which would make it hard to do manually. I'm not sure how to proceed...